types/classificator/index.d.ts

```diff
@@ -0,0 +1,120 @@
+// Type definitions for classificator 0.3
+// Project: https://github.com/Wozacosta/classificator
+// Definitions by: Nixinova <https://github.com/Nixinova>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export as namespace classificator;
+
+export = Classificator;
+
+declare function Classificator(options?: Classificator.Options): Classificator.NaiveBayes;
+
+declare namespace Classificator {
+    interface ClassificationResults {
+        likelihoods: Array<{
+            category: string,
+            logLikelihood: number,
+            logProba: number,
+            proba: number,
+        }>;
+        predictedCategory: string;
+    }
+
+    interface Options {
+        /**
+         * Given an input string, tokenize it into an array of word tokens.
+         *
+         * By default, removes punctuation and splits on spaces.
+         */
+        tokenizer?: (text: string) => string[];
+
+        /**
+         * Alpha parameter of the additive smoothing operation.
+         *
+         * Defaults to `1`.
+         */
+        alpha?: number;
+
+        /**
+         * Defines how the prior probablity is calculated.
+         *
+         * If set to false, the classifier will use an uniform prior rather than a learnt one.
+         *
+         * Defaults to `true`.
+         */
+        fitPrior?: boolean;
+    }
+
+    /**
+     * Naive-Bayes Classifier
+     *
+     * This is a naive-bayes classifier that uses Laplace Smoothing.
+     *
+     * Takes an (optional) options object containing:
+     * - `tokenizer` => custom tokenization function
+     */
+    interface NaiveBayes {
+        (options?: Options): NaiveBayes;
```

Comment:
```ts
        (options?: Options): NaiveBayes;
```
are you sure this is correct?
```js
const bayes = require("classificator");
const classifier = bayes();
classifier({});
```
this will just fail in JS